tldw_Server_API/app/core/DB_Management/migrations_v6_evaluation_recipes.py

```
@@ -0,0 +1,84 @@
+                    with contextlib.suppress(sqlite3.Error):
+                        cursor.execute(column_sql)
+
+            with contextlib.suppress(sqlite3.Error):
+                cursor.execute(
+                    "UPDATE evaluation_recipe_runs SET updated_at = COALESCE(updated_at, CURRENT_TIMESTAMP)"
+                )
+
+            conn.commit()
+            logger.info("Applied evaluation recipe schema migration")
+            return True
+    except Exception as exc:  # pragma: no cover - defensive migration wrapper
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)